### DISCUSSION TITLE
Colours are washed out when using a VAE (Any Model)

### DISCUSSION THREAD

**Discussion by StanleyPain** [vladmandic/sdnext#3203] [Q&A]
Did I mess up a setting some where? I'm getting washed out colours.

**Comment by mkossigit** [comment]
Channels Last option in compute solved this for me.. was mentioned on the discord.
i messed around with vae upcast, preferred vae variant, live preview method.. all did nothing.

Also selecting none in vae model, and restarting helps.

**Reply by StanleyPain** [reply]
Thanks for the info!

Enabling channels last or disabling it?

**Reply by StanleyPain** [reply]
Tried it with the setting on and off but it didn't make a difference. The strange part is that it was working properly for weeks.
